### PR TITLE
feat: Add country preference

### LINKED ISSUE #1050: Ability to set the default country and currency
### Feature Request

**Feature Description**
Just like the event brite app, there can be a preference set by the user to set the default country and currency.
Initially a new fragment can be created with the name Event Settings and subsequently other features which are in the process of implementation (check-in restrictions, auto check in) can be added to it.

**Would you like to work on the issue?**
Yes.

**Screenshots**
whatsapp image 2018-06-09 at 18 57 03 1

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1050 

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream development branch.

Screenshots for the change:
ezgif com-resize 21

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Rather than having single property in nested screen, show both properties in parent scree

**Comment by sridharjajoo** [comment]
@iamareebjamal The Payment preference uses the **Preference screen** in its layout whereas for implementing the country spinner, a normal XML layout has been used as spinners can't be included in the **Preference Screen**. So, implementing them on a single screen will only be possible if I remove the implementation of the **Preference Screen** for Payments and handle the shared preference programmatically. Should I continue with this approach?

**Comment by iamareebjamal** [comment]
AFAIK preference dialog can be used for country

**Comment by sridharjajoo** [comment]
@iamareebjamal @mishuvs@Masquerade0097  I have made the changes and removed the extra screens. Kindly review. The screenshot has been updated. Thank you.

File: app/src/main/java/org/fossasia/openevent/app/core/settings/EventSettingsFragment.java [hunk 1]
```diff
@@ -50,4 +29,20 @@ public void onResume() {
         ViewUtils.setTitle(this, getString(R.string.settings));
```
Comment (mishuvs) [inline app/src/main/java/org/fossasia/openevent/app/core/settings/EventSettingsFragment.java]: Should be "Event Settings"

**Comment by sridharjajoo** [comment]
@mishuvs I have made the required changes. Kindly check the Screenshot.

**Pull request merged**